docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1729,6 +1729,16 @@ The limit protects the system’s stability from potential abuse or mistakes. To
 Invalid series are skipped during the ingestion, and valid series within the same request are ingested.
 {{< /admonition >}}
 
+### err-mimir-max-label-names-per-info-series
+
+This non-critical error occurs when Mimir receives a write request that contains an info series with a number of labels that exceed the configured limit.
+An info series is a series where the metric name ends in `_info`.
+The limit protects the system’s stability from potential abuse or mistakes. To configure the limit on a per-tenant basis, use the `-validation.max-label-names-per-info-series` option.
```

Comment:
Nit: I think `misuse` has a more positive connotation than `abuse`.

Reply:
The doc uses `abuse` 21 times, and `misuse` zero times. Let's change in a different PR if we want to.